eyelid-match-ops/src/primitives/poly/test/inv.rs

```diff
@@ -46,7 +51,13 @@ fn inverse_test_helper<const MAX_POLY_DEGREE: usize>(f: &Poly<MAX_POLY_DEGREE>)
 #[test]
 fn test_inverse_with_small_random_coefficients() {
```

Comment:
Since this test is now testing key generation as well, if we get failures, it helps to rename it:

```suggestion
fn test_key_generation_and_inverse() {
```